whylog/regexassistant/__init__.py

```diff
@@ -0,0 +1,102 @@
+class RegexAssistant(object):
+    """
+    Responsible for creating regex for each line of Rule.
+    RegexAssistant helps user to write regex, it can also proposes regex.
+    One RegexAssistant per one entering Rule.
+    """
+
+    def _obvious_regex(self, text):
+        """
+        Creates regex form text by simple transformation:
+        - add backslash to special characters
+        - text -> '^' + text + '$'
+        :param text:
+        :return:
+        """
+        # not finished yet
+        return '^' + text + '$'
+
+    def __init__(self, line_objects=tuple()):
+        self.regexes = dict(
+            [
+                (line, self._obvious_regex(line.line_content)) for line in line_objects
+            ]
+        )
+        self.history = dict([(line, []) for line in line_objects])
+        pass
```

Comment:
this statement does nothing. `pass` should only be used to prevent syntax error of empty identation block